packages/minifier-js/minifier.js

```diff
@@ -7,13 +7,18 @@ meteorJsMinify = function (source) {
   terser = terser || Npm.require("terser");
 
   try {
+    source = source
+      .replace(/Meteor\.isServer|Meteor\.isDevelopment|process\.env\.NODE_DEBUG/g, 'UGLYFYJS_FALSE')
+      .replace(/Meteor\.isClient|Meteor\.isProduction/g, 'UGLYFYJS_TRUE');
```

Comment:
You should actually just be able to put these expressions in the `global_defs` map below, so you don't have to use a regular expression, and the minifier can replace the actual expressions in the AST—much safer!